### PR TITLE
[docs] Fix markdown in profiling doc page

### PR DESCRIPTION
**Pull request by oleksii-lisovyi**
## The Issue
Here's how steps numbers section looks a bit broken:
Screenshot 2023-04-05 at 19 47 25

## How This PR Solves The Issue
Well, it looks now as initended.

<a href="https://gitpod.io/#https://github.com/ddev/ddev/pull/4806"></a>

### CODE REVIEW

File: docs/content/users/debugging-profiling/blackfire-profiling.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ The Blackfire CLI is built into the web container, so you don’t need to instal
     ```
 
   You can also do this with `ddev config global --web-environment-add="BLACKFIRE_SERVER_ID=<id>,BLACKFIRE_SERVER_TOKEN=<token>,BLACKFIRE_CLIENT_ID=<id>,BLACKFIRE_CLIENT_TOKEN=<token>"`, but any existing environment variables will be deleted.
```
Comment (mattstein) [inline docs/content/users/debugging-profiling/blackfire-profiling.md:30]:
```suggestion
    You can also do this with `ddev config global --web-environment-add="BLACKFIRE_SERVER_ID=<id>,BLACKFIRE_SERVER_TOKEN=<token>,BLACKFIRE_CLIENT_ID=<id>,BLACKFIRE_CLIENT_TOKEN=<token>"`, but any existing environment variables will be deleted.
```

**Review by mattstein — CHANGES_REQUESTED** [review]
Thank you for this, @oleksii-lisovyi! It was definitely mangled and you definitely fixed it.

Would you mind adding two spaces to fully indent that “You can also (...)” sentence with its parent list item?

Screen Shot 2023-04-05 at 10 06 14 AM@2x

I left that as a code suggestion you can commit using **Commit suggestion**.

**Comment by rfay** [comment]
Thanks @oleksii-lisovyi - Could you also say something about your blackfire usage? I was wondering if anybody used it any more since the free/hacker plan went away. I assume you use Platform.sh?

**Comment by oleksii-lisovyi** [comment]
@rfay honestly just started exploring this in depth. haven't had any decent experience to be able to comment on that.
so far trying to make it work, tho there are some limitations with license I wasn't yet able to overcome.

**Comment by rfay** [comment]
Yeah, I don't think you can use it at all any more without a paid account.

**Comment by tyler36** [comment]
From Blackfire [pricing page](https://www.blackfire.io/pricing/#free-trial):

> We don't provide a free trial. 

If it's paid-only, perhaps it should be extracted to an addon and removed from DDEV?

**Comment by rfay** [comment]
Right, there is no free trial, but I'm interested if people have a subscription and are still using it any more. Blackfire is a product of Platform.sh, a major sponsor of DDEV, but no matter how much I pester, I can't get them to bring back the hacker/localdev/free account.

Moving it to an add-on seems possible.

**Pull request merged**